server/src/main/scala/za/co/absa/atum/server/api/http/Routes.scala

```diff
@@ -149,16 +169,16 @@ trait Routes extends Endpoints with ServerOptions {
   }
 
   private def createServerEndpoint[I, E, O](
-    endpoint: PublicEndpoint[I, E, O, Any],
-    logic: I => ZIO[HttpEnv.Env, E, O]
-  ): ZServerEndpoint[HttpEnv.Env, Any] = {
+                                             endpoint: PublicEndpoint[I, E, O, Any],
```

Comment:
I like the previous formatting more :( but we decided not to insist on anything specific so I'll obey 😆 

Anyway, do you use some formatter, in IntelliJ for example?

Reply:
Yes, but I have reverted it back to normal